foxy/distribution.yaml

```diff
@@ -3070,6 +3070,16 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: foxy-devel
     status: maintained
+  performance_test-release:
+    release:
```

Comment:
could you add a source and a doc entry?

Reply:
@ivanpauno I have pushed my changes. Thank you!